etree.go

```diff
@@ -314,15 +327,23 @@ func (e *Element) Text() string {
 	}
 
 	text := ""
+L:
 	for _, ch := range e.Child {
-		if cd, ok := ch.(*CharData); ok {
+		switch cd := ch.(type) {
+		case *CharData:
 			if text == "" {
 				text = cd.Data
 			} else {
 				text = text + cd.Data
 			}
-		} else {
-			break
+		case *PlainTextData:
+			if text == "" {
+				text = cd.Data
+			} else {
+				text = text + cd.Data
+			}
+		default:
+			break L
```

Comment:
With a single `CharData` type, the switch statement should no longer be necessary.